### DISCUSSION TITLE
GL路由器siflower的芯片无ipk

### DISCUSSION THREAD

**Discussion by fanqiangdatui** [messense/aliyundrive-webdav#319] [Q&A] [answered]
似乎没有siflower的固件，虽然都是mips，但是不通用

**Comment by messense** [comment, marked as answer]
试试这个。
[aliyundrive-webdav_1.8.4_mips_siflower.ipk.zip](https://github.com/messense/aliyundrive-webdav/files/9173345/aliyundrive-webdav_1.8.4_mips_siflower.ipk.zip)
